### DISCUSSION TITLE
Thought Provoking Stories on New Page

### DISCUSSION THREAD

**Discussion by Heinrich-XIAO** [zen-browser/desktop#3094] [Polls]
TL;DR Vote no.

I feel like the stories on the front page are somewhat confusing to remove. The first thought was that it was probably pocket, but `extensions.pocket.enabled` was already `false`. I finally realized it was in `browser.newtabpage.activity-stream.feeds.section.topstories`, but that made me thinking, is having a news feed really that zen? Zen is about enlightenment and meditation. Is modern news meditative?

**Comment by Enrok** [comment]
I'd like to have a customizable new tab. I hate 'research bar' tab and the problems of you typing a valid url and the browser jumping to it when all you wanted was doing a research (but i reckon that's a marginal behavior)
